scripts/train-t2m-onnx-v4.py

```
@@ -0,0 +1,376 @@
+
+    # Per-action MEDOID exemplar: the real window closest to the action's mean
+    # in 6D space. z=0 (the app's inference latent) is supervised toward THIS
+    # instead of each individual sample — a conditional-mean target averages
+    # phase-misaligned windows into mush, while the medoid is a crisp real
+    # clip. z~posterior keeps per-sample reconstruction (variety).
+    act_idx = torch.tensor(tk.argmax(1))
+    flat = R6.reshape(N, -1)
+    medoid6 = torch.zeros(V, T, D6)
+    for aI in range(V):
+        m_ = (act_idx == aI).nonzero(as_tuple=True)[0]
+        if not len(m_): continue
```

Ruff (0.15.20)

[error] Multiple statements on one line (colon)

(E701)